src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj

```diff
@@ -134,6 +134,19 @@
     </ItemGroup>
     <Error Text="Unexpected System package(s) @(_secondarySystemPackages)" Condition="'@(_secondarySystemPackages)' != ''" />
 
+    <ItemGroup>
+      <!-- pick up any src our sources directory from packages contributing files -->
+      <_sourcePaths Include="@(FilesToPackage->'$(PackagesDir)\%(NuGetPackageId)\%(NuGetPackageVersion)\sources')" />
+      <_sourcePaths Include="@(FilesToPackage->'$(PackagesDir)\%(NuGetPackageId)\%(NuGetPackageVersion)\src')" />
+
+      <!-- include all source from those directories -->
+      <_sourceFile Condition="'%(_sourcePaths.Identity)' != ''" Include="%(_sourcePaths.Identity)\**\*.*" />
+      <FilesToPackage Include="@(_sourceFile)">
+        <TargetPath>src\%(RecursiveDir)%(FileName)%(Extension)</TargetPath>
+        <IsSourceCodeFile>true</IsSourceCodeFile>
+      </FilesToPackage>
+    </ItemGroup>
```

Comment:
Yeah, this isn't the first place we're duplicating so I decided not to tackle that yet, but I did make sure the change was identical so that deduping later will be easy.